plugin-machine/che-plugin-machine-ext-client/src/main/resources/org/eclipse/che/ide/extension/machine/client/MachineLocalizationConstant.properties

```diff
@@ -150,3 +150,6 @@ failed.to.create.recipe=Failed to create recipe
 failed.to.save.recipe=Failed to save recipe
 failed.to.find.machine=Failed to find machine {0}
 failed.to.get.processes=Failed to get processes from machine {0}
+
+ssh.connect.info=You can connect your SSH client to the machine ''{0}'' by this command:\
+  \nssh user@{1} -p {2} [-i /path/to/your/ssh]
```

Comment:
I think we should write something like [-i /path/to/your/ssh/key] or [-i /path/to/your/key]